sparse_strips/vello_cpu/src/render.rs

```diff
@@ -196,6 +201,16 @@ impl RenderContext {
         self.paint = paint.into();
     }
 
+    /// Set the current paint transform.
```

Comment:
Some more complete docs here might be good. Even just mentioning the use case, noting that the brush is impacted by the global transform, etc. (that is, that this is logically applied "after" the current global transform).

I won't block on this